prdoc/pr_3374.prdoc

```diff
@@ -0,0 +1,14 @@
+# Schema: Polkadot SDK PRDoc Schema (prdoc) v1.0.0
+# See doc at https://raw.githubusercontent.com/paritytech/polkadot-sdk/master/prdoc/schema_user.json
+
+title: removed `pallet::getter` from `pallet-timestamp`
+
+doc:
+  - audience: Runtime Dev
+    description: |
+      This PR removes all the `pallet::getter` usages from `pallet-timestamp`, and updates depdendant runtimes accordingly.
+      The syntax `StorageItem::<T, I>::get()` should be used instead.
+
+crates: 
+  - name: pallet-timestamp
+  - name: pallet-election-provider-multi-phase
```

Comment:
why anyone using `pallet-election-provider-multi-phas` needs to care about this PR?

Reply:
In their [test-staking-e2e](https://github.com/paritytech/polkadot-sdk/blob/master/substrate/frame/election-provider-multi-phase/test-staking-e2e/src/lib.rs#L49), they call the timestamp pallet.

Reply:
The test crate is not released.